presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -480,4 +500,60 @@ public String toString()
                     .toString();
         }
     }
+
+    private static class TimestampAndPrecision
+    {
+        private final int id;
+        private final int precision;
+        private final String value;
+        private final String roundedValue;
+
+        public TimestampAndPrecision(int id, int precision, String value, String roundedValue)
+        {
+            this.id = id;
+            this.precision = precision;
+            this.value = value;
+            this.roundedValue = roundedValue;
+        }
+
+        public int getId()
+        {
+            return id;
+        }
+
+        public String getPrecision()
+        {
+            switch (precision) {
+                case 3: return "MILLIS";
```

Comment:
put return on the next line (the code should be immune to reformat action in correctly configured IDE)